src/cephadm/cephadm

```diff
@@ -7322,8 +7322,15 @@ def command_maintenance(ctx: CephadmContext) -> str:
 
 def _get_parser():
     # type: () -> argparse.ArgumentParser
+    description = """:program:`cephadm` is a command line tool to manage the local host for the cephadm orchestrator.
+
+It provides commands to investigate and modify the state of the current host.
```

Comment:
```suggestion
It provides commands to investigate and modify ceph containers on the current host.
```

(nit)  reword? the prior verbiage somewhat implies a more general host mgmt tool

Reply:
we have `gather-facts` and `networks`, so we're doing more that containers.